Retail.lua

```diff
@@ -424,55 +467,67 @@ addon.Spells = {
         [114052] = { type = BUFF_DEFENSIVE, parent = 114050 }, -- Ascendance (Restoration)
     [118345] = { type = CROWD_CONTROL }, -- Pulverize
     [118905] = { type = CROWD_CONTROL }, -- Static Charge
-    [191634] = { type = BUFF_OFFENSIVE }, -- Stormkeeper
+    [191634] = { type = BUFF_OFFENSIVE }, -- Stormkeeper (Ele)
+    [320137] = { type = BUFF_OFFENSIVE }, -- Stormkeeper (Enh)
```

Comment:
```suggestion
    [320137] = { type = BUFF_OFFENSIVE, parent = 191634 }, -- Stormkeeper (Enh)
```

Reply:
I don't really mind but Enh and Ele Stormkeeper _are_ different spells with different functionalities. I'm pretty sure I heard that Enh Stormkeeper is kinda weak/irrelevant (at least in PvP) whilst the Ele Stormkeeper is kind of a big deal, and this balance might shift any time, so I think you might want to display one but not the other.